src/device/x86-linux/gpioctl_x86_linux.cpp

```diff
@@ -13,25 +13,128 @@
  * limitations under the License.
  */
 
-#if defined(__LINUX__)
-
 #include "iotjs_def.h"
 #include "iotjs_objectwrap.h"
 #include "iotjs_module_gpioctl.h"
 
+//
+// this is dummy code for x86.
+// please add new 'device' file for your purpose.
+// use timer with 1msec delay to implement async callback.
+// real gpio implementation would be better to use threading
+//
 
 namespace iotjs {
 
-/*
- * Use default dummy for linux-x86
- */
+
+// type for delayed callback
+typedef enum {
+  DELAYEDCALL_SETSPIN = 1,
+} dc_type;
```

Comment:
Type name should be UpperCamelCase
https://github.com/Samsung/iotjs/wiki/Coding_Style_Guideline#type-names